point-of-sale-app/ui/public/404.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
```

Comment:
Praise: Oooo! 👀 This is a really cool 404 page. LOOL.

Reply:
Hahaha. I haven't used it anywhere. Maybe I should add it back again!